src/test/java/domain/LineTest.java

```diff
@@ -12,39 +12,31 @@ public class LineTest {
 
 
     @Test
-    @DisplayName("")
+    @DisplayName("Line 생성 확인")
     void line() {
```

Comment:
DisplayName을 적는게 헷갈리신다면 다음과 같이 지정할수도 있습니다 😊

```suggestion
    void Line_생성_확인() {
```

Reply:
감사합니당